src/microphone/const.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Default sample rate for microphone streams.
+ * @public
+ */
+export const DefaultSampleRate = 16000
+
+/**
+ * Error to be thrown when an action occurred before the microphone was initialized.
+ * @public
+ */
+export const ErrNotInitialized = new Error('Microphone is not initialized')
+
+/**
+ * Error to be thrown when the initialize method of a Microphone instance is called more than once.
+ * @public
+ */
+export const ErrAlreadyInitialized = new Error('Microphone is already initialized')
+
+/**
+ * Error to be thrown when the device does not support the Microphone instance's target audio APIs.
+ * @public
+ */
+export const ErrNoBrowserSupport = new Error('Current browser does not support audio API')
```

Comment:
Since the microphone will no longer be browser-only, this error should be changed to reflect that and the error dispatching code in the Client updated accordingly:
```suggestion
export const ErrDeviceNotSupported = new Error('Current device does not support microphone API')
```